app/domain/logs.rb

```diff
@@ -629,6 +629,20 @@ module AuthnJwt
         code: "CONJ00124I"
       )
 
+      ParsingMappedClaims = ::Util::TrackableLogMessageClass.new(
+        msg: "Parsing mapped claims value '{0-mapped-claims}'",
+        code: "CONJ00125D"
+      )
+
+      ParsedMappedClaims = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully parsed mapped claims '{0-mapped-claims-table}'",
+        code: "CONJ00126D"
+      )
+
+      ClaimMapDefinition = ::Util::TrackableLogMessageClass.new(
+        msg: "Mapping annotation name '{0-annotation-value}' to the claim name '{1-claim-name}'",
```

Comment:
@hilagross @shulifink 
Please review

Reply:
when do we call this one? 
I need have context.

Reply:
It will be printed for each mapping tuple at debug level